src/python_testing/TC_DA_1_7.py

```diff
@@ -64,14 +64,40 @@ def extract_akid(cert: Certificate) -> Optional[bytes]:
 
 
 class TC_DA_1_7(MatterBaseTest):
+    ''' TC-DA-1.7
+
+        This test requires two instances of the DUT with the same PID/VID to confirm that the individual
+        devices are provisioned with different device attestation keys even in the same product line.
+
+        In order to fully test this locally, it requires two separate instances of the example app
+        with different keys, different discrimimators, different ports, and different KVSs.
+        The example apps use the SDK authority key, so the script needs the allow_sdk_dac arg set.
+
+        One suggested method of testing:
+
+        Terminal 1: (first DUT - default port 5540)
+        rm kvsa
+        <path_to_all_clusters>/chip-all-clusters-app --dac_provider \
+            credentials/development/commissioner_dut/struct_cd_authorized_paa_list_count1_valid/test_case_vector.json \
+            --product-id 32768 --KVS kvsa --discriminator 12
+
+        Terminal 2: (second DUT - port 5541)
+        rm kvsb
+        <path_to_all_clusters>/chip-all-clusters-app --dac_provider \
```

Comment:
Suggest `${PATH_TO_ALL_CLUSTERS}` instead of `<path_to_all_clusters>` --> helps with reusing on command line by copying this